langchain-core/src/language_models/chat_models.ts

```diff
@@ -359,7 +388,7 @@ export abstract class BaseChatModel<
       handledOptions.tags,
       this.tags,
       handledOptions.metadata,
-      this.metadata,
+      inheritableMetadata,
```

Comment:
this is the non inheritable metadata I think?

Reply:
and in py I think we put it as inheritable

Reply:
You are correct. Fixed in recent commit, thanks